<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShotStatusWidget</class>
 <widget class="QWidget" name="ShotStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="shotStatusTreeWidget">
       <column>
        <property name="text">
         <string>Sequence</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Shot</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artist</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reportBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Generate Report</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
